src/win/tty.c

```diff
@@ -2384,3 +2379,16 @@ static void uv__tty_console_signal_resize(void) {
     uv_mutex_unlock(&uv__tty_console_resize_mutex);
   }
 }
+
+void uv_tty_set_vterm_state(uv_tty_vtermstate_t state) {
+  uv_sem_wait(&uv_tty_output_lock);
+  uv__vterm_state = state;
```

Comment:
uv__need_check_vterm_state
```suggestion
  uv__need_check_vterm_state = FALSE;
  uv__vterm_state = state;
```